ihmc-footstep-planning/src/main/java/us/ihmc/footstepPlanning/log/FootstepPlannerLogLoader.java

```diff
@@ -186,7 +184,7 @@ public LoadResult load(File logDirectory)
          {
             InputStream bodyPathParametersPacketInputStream = new FileInputStream(bodyPathParametersFile);
             jsonNode = objectMapper.readTree(bodyPathParametersPacketInputStream);
-            log.getBodyPathParametersPacket().set(bodyPathParametersSerializer.deserialize(jsonNode.toString()));
+            log.getAStarBodyPathPlannerParametersPacket().set(bodyPathParametersSerializer.deserialize(jsonNode.toString()));
```

Comment:
Not sure about the `AStar` convention for the BodyPathPlannerParameters, it makes it pretty long and I could drop it @PotatoPeeler3000

Reply:
Yep agreed will do